src/Oceanostics.jl

```diff
@@ -71,6 +71,28 @@ function add_background_fields(model)
 end
 #---
 
+#+++ Utils for removing mean velocities
+using Oceananigans.Biogeochemistry: biogeochemical_auxiliary_fields
+"""
+    $(SIGNATURES)
+
+Remove mean velocities from the model resolved velocities.
+"""
+function remove_mean_velocities(model, mean_velocities)
```

Comment:
I think the name of this function is a bit misleading since it doesn't just _remove mean velocities_, it returns a `models`'s fields minus the `mean_velocities`.

I think we should either:

1. Extend Oceananigan's `fields` function with `from Oceananigans import: fields`, and then replace `remove_mean_velocities` with `fields`, therefore adding one method to it.
2. Call it something else that's descriptive, such as `fields_without_mean()`

If we do the latter we might change the call to `fields_without_means(model, args...)`, so that we can remove the mean from any field (`u`, `v`, `w`, `b`, `T`, etc).

Reply:
I will go with the second suggestion here. But I'm not sure how to implement variable number of arguments in this case. Currently the function takes one `NamedTuple` containing mean velocities. Any psuedo-code example?

Reply:
This is working test example that should give you all the tools you need:

```julia
julia> function test(model; kwargs...)
           for (variable, mean) in kwargs
              @info "Here is where you you remive the $mean from $variable in $model"
           end
       end
test (generic function with 1 method)

julia> test("bla", a=4)
[ Info: Here is where you you remive the 4 from a in bla
```